src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Crypto.cs

```diff
@@ -2,9 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
```

Comment:
can you add README.md in native/libs/S.S.C.Native which gives the instructions for debugging? I think this is super useful